client/src/ui/DeckBuilderScreen.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="_home">
```

Comment:
Nit: _home -> _deck-builder-screen

(the CSS scoping system is great but can still have conflicts between the parent and the root element of the component, so I try to name the root element with a globally-unique-ish class)